test/jasmine/tests/plots_test.js

```diff
@@ -153,26 +153,6 @@ describe('Test Plots', function() {
 
             testSanitizeMarginsHasBeenCalledOnlyOnce(gd);
         });
-
-        it('should sort base plot modules on fullLayout object', function() {
```

Comment:
It's great that we don't need to sort anymore, but what was the issue that forced us to sort before? Can we do something to test that the underlying issue has been solved?

Reply:
It's already covered in https://rreusser.github.io/plotly-mock-viewer/#splom_with-cartesian where the blue scattergl pt in the lower-left quadrant sits on top of the orange splom trace.

There's also:

https://github.com/plotly/plotly.js/blob/9772ef6dd5ec5417ab50dd378aaa536a8c845336/test/jasmine/tests/splom_test.js#L1063-L1099